README.md

```diff
@@ -32,12 +32,12 @@ All development and testing were done on **linux** OS (mostly ubuntu). For the w
 
 #### Install & run CPU version
 
-**Requirements:** Python3 (numpy,matplotlib) & C/C++ compiler (g++,make)
+**Requirements:** Python3 (numpy,matplotlib) & C/C++ compiler (g++,make); SDL
 
 ##### First run: Graphene with point-charges
  1. clone the repository: `clone https://github.com/Probe-Particle/ProbeParticleModel.git`
```

Comment:
Hi, thanks for the feedback and sorry that I missed the meeting yesterday, I was confused about time zones. Anyway, how is `pip` supposed to work here? I definitely did not manage to get it working for me, but that should be probably discussed elsewhere. (Should I start a new issue about it or can we try to resolve it in direct email communication?) Anyway, regarding what should go into README.md: You seem to have a typo in `pip install -e ppfam`, it should be either `ppafm`, but perhaps just `pip install -e .` (as the file `pyproject.toml` is in the root directory `ProbeParticleModel`, not in `ProbeParticleMode/ppafm`).

Reply:
> Anyway, how is `pip` supposed to work here? I definitely did not manage to get it working for me, but that should be probably discussed elsewhere.

Pip will install python dependencies. Since #92 has been merged, all the issues should be fixed. Please, try to pull the latest changes from the `main` branch and do `pip install -e.`  from the root directory.

 > Should I start a new issue about it or can we try to resolve it in direct email communication?

I think all repository-related communications should better be kept in the repository. This will simplify our life in the future if we want to find out reasons for some choices.

> as the file `pyproject.toml` is in the root directory `ProbeParticleModel`, not in `ProbeParticleMode/ppafm`

Please keep in mind that the root directory has been renamed to `ppafm`.

> You seem to have a typo in `pip install -e ppfam`

Yes, fixed that.